sdk/storage/Azure.Storage.Blobs/tests/VersionLevelWormTests.cs

```diff
@@ -0,0 +1,899 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage.Sas;
+using Azure.Storage.Test;
+using Azure.Storage.Test.Shared;
+using Microsoft.Azure.Management.Storage;
+using Microsoft.Identity.Client;
+using Microsoft.Rest;
+using NUnit.Framework;
+
+namespace Azure.Storage.Blobs.Test
+{
+    //[PlaybackOnly("https://github.com/Azure/azure-sdk-for-net/issues/19575")]
+    public class VersionLevelWormTests : BlobTestBase
+    {
+        public VersionLevelWormTests(bool async, BlobClientOptions.ServiceVersion serviceVersion)
+            : base(async, serviceVersion, null /* RecordedTestMode.Record /* to re-record */)
+        {
+        }
+
+        [Test]
+        [ServiceVersion(Min = BlobClientOptions.ServiceVersion.V2020_06_12)]
+        public async Task SetImmutibilityPolicyAsync()
+        {
+            // Arrange
+            await using DisposingVersionLevelWormContainer vlwContainer = await GetTestVersionLevelWormContainer(TestConfigOAuth);
```

Comment:
can we not use `vlw` or `VersionLevelWorm` in tests either? We should use official naming everywhere. In few years nobody will remember what `vlw` is.

Reply:
Fixed.